docs/pages/machine-id/guides/github-actions.mdx

```diff
@@ -40,17 +40,17 @@ repo, however this value should be replaced with your own unique repo.
 
 ## Step 2/3. Create your Machine ID bot
 
-With the join token for GitHub Actions created, the next step is to create the Machine ID bot,
-and ensure that it is configured to use the newly created token.
+With the join token created, the next step is to create the Machine ID bot that
+the token will grant access to.
 
-The Machine ID bot created in this example will be used to access a specific node on the cluster
-via `tsh ssh`, and will therefore require a role that can access the cluster as needed. This example
-configuration will apply the `access` role, however care should be taken to either create or apply
-a role according to the principle of least privilege in production environments. Additionally, it
-should have explicit access to the cluster using a username created specifically for the bot user alone.
+In this example, the bot is given the default `access` role. In a production
+environment, the principle of least privilege should be applied and you should
+create a role that grants the bot access to the precise resources that will be
+needed in your CI/CD workflow.
 
-Enter the following command from your workstation with `tsh` access, replacing the `username` value with
-a Linux user on the host that you want your GitHub Actions flow to be able to connect to:
+Enter the following command from your workstation with `tsh` access, replacing
```

Comment:
Why do we say _with tsh access_ and then list a `tctl` command?

Reply:
It's a weird one, I based this on the rest of our documentation which I think is trying to convey that you should have `tsh` authenticated so you can use `tctl`. I'll see if I can come up with some better wording.

Reply:
Resolved